api/src/main/resources/workbench.yaml

```diff
@@ -1253,6 +1266,40 @@ definitions:
         type: integer
         format: int64
 
+  CdrVersionListResponse:
+    type: object
+    required:
+      - items
+      - defaultCdrVersionId
+    properties:
+      items:
+        type: "array"
+        items:
+          $ref: "#/definitions/CdrVersion"
+      defaultCdrVersionId:
+        type: "string"
```

Comment:
Should this be a long, since it's a long in the db?
type: integer
format: int64